lib/account-tasks/helpers/activateAndVoteHelper.ts

```diff
@@ -2,16 +2,21 @@ import chalk from "chalk";
 import { Signer } from "ethers";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
-export async function activateAndVote(hre: HardhatRuntimeEnvironment, signer: Signer) {
+export async function activateAndVote(
+  hre: HardhatRuntimeEnvironment,
+  signer: Signer,
+  verboseLog: boolean
+) {
   const accountContract = await hre.ethers.getContract("Account");
   const managerContract = await hre.ethers.getContract("Manager");
 
   const electionWrapper = await hre.kit.contracts.getElection();
   const electionContract = await hre.ethers.getContractAt("IElection", electionWrapper.address);
 
   const groupList = await managerContract.getGroups();
-  console.log("groups:", groupList);
-
+  if (verboseLog) {
```

Comment:
Instead of having these conditions. Could we introduce logging class that would have something like

log
logVerbose
setLevel

and at the beginning of the request we would set the level of logging ?